docs/src/main/paradox/slick.md

```diff
@@ -37,7 +37,7 @@ The offset is stored in the same transaction as the `DBIO` returned from the `ha
 processing semantics if the projection is restarted from previously stored offset.
 
 Scala
-:  @@snip [SlickProjectionDocExample.scala](/examples/src/test/scala/docs/slick/SlickProjectionDocExample.scala) { #projection-imports #actor-system #exactlyOnce }
+:  @@snip [SlickProjectionDocExample.scala](/examples/src/test/scala/docs/slick/SlickProjectionDocExample.scala) { #projection-imports #exactlyOnce }
```

Comment:
better to not show ActorSystem creation in doc examples because users will copy paste and suddenly they have started many ActorSystems